gluoncv/auto/tasks/image_classification.py

```diff
@@ -97,6 +111,39 @@ def _train_image_classification(args, reporter):
         num_trials = args.pop('num_trials')
     except AttributeError:
         task = None
+
+    # mxnet and torch dispatcher
+    dispatcher = None
+    torch_model_list = None
+    mxnet_model_list = list(get_model_list())
```

Comment:
requires mxnet here?

Reply:
I think get_model_list() is part of gluoncv api and not require mxnet?

Reply:
no, model zoo still requires mxnet, only auto module can optionally use mxnet, otherwise gluoncv.torch is the right module